code/modules/power/fusion/kinetic_harvester.dm

```diff
@@ -1,3 +1,4 @@
+// This machine is COMPLETELY BROKEN due to material and reagent reworks, TODO: total overhaul
```

Comment:
is it tho, i thought you fixed it in another PR

Reply:
It's not fixed on dev yet.